### PR TITLE
Pbi 279543 landing page consistency

### PR DESCRIPTION
**Pull request by jasnames**

### CODE REVIEW

File: content/visualize-data/power-bi-connector.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ Use Microsoft Power BI to edit the query generated from the connector to modify
         | `RangeEnd` | Description: `<optional>`<br> Required: `selected`<br> Type: `Date/Time`<br> Suggested Values: `<Any value>, <List of values>, <Query>`<br> Current Value: `<End date of the date range>` |
      
        **Note:** `RangeStart` and `RangeEnd` must be named and mixed-cased as is for incremental refresh to work. Type must always be `Date/Time`.
-
+   <!-- Jason Ames 12/14/21 - Something about this final indent breaks this display and puts the table and note in a code box. Maybe this many indent levels aren't supported?-->
```
Comment (afloresOSI) [inline content/visualize-data/power-bi-connector.md:131]: @rmuolic Can you take a look at this display issue in the Power BI topic?

**Review by afloresOSI — CHANGES_REQUESTED** [review]

File: content/visualize-data/visualize-data-overview.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ uid: lpvisualizedata
 
 # Visualize data
 
-Use the Trend page to monitor assets, anticipate problems, and proactively perform preventative maintenance.
+Use the `Trend` page to monitor assets, anticipate problems, and proactively perform preventative maintenance.
 
-On the Trend page, you can convert stream data to a graphic view, which can reveal patterns in data, data anomalies, or trouble spots. 
+On the `Trend` page, you can convert stream data to a graphic view, which can reveal patterns in data, data anomalies, or trouble spots. 
+<!-- Jason Ames 12/14/21 - This overview says nothing about the Power BI Connector and only addresses Trend, which has it's own landing page already. A more generic intro that actually mentions visualizing data might be helpful?-->
```
Comment (afloresOSI) [inline content/visualize-data/visualize-data-overview.md:10]: @rmuolic Do you have updates planned for this?

File: content/overview/get-started/gs-clients.md [hunk 3]
```diff
@@ -4,8 +4,9 @@ uid: gsClients
 
 # Get started with clients
 
-Clients provide a way for users or applications to authenticate against OCS from applications other than the portal. OCS supports different client types, each of which support different types of applications. For more information on client types, see the following topics:
- 
+Clients provide a way for users or applications to authenticate against OCS from applications other than the portal. OCS supports different client types, each of which support different types of applications:
+<!--Jason Ames 12/13/21 Since we haven't otherwise listed the names of the client types, I'd prefer to leave the bullets in here even though the links are redundant to the Related Links.-->
```
[the remark this answers is not in the document]
Comment (jasnames) [reply content/overview/get-started/gs-clients.md]: done

File: content/visualize-data/visualize-data-overview.md (hunk 2 above)
Comment (rmuolic) [reply content/visualize-data/visualize-data-overview.md:10]: > @rmuolic Do you have updates planned for this?

@afloresOSI - Yes, I'm planning on implementing conceptual content in an upcoming sprint, see [PBI 265023](https://dev.azure.com/osieng/engineering/_workitems/edit/265023).

File: content/visualize-data/power-bi-connector.md (hunk 1 above)
Comment (rmuolic) [reply content/visualize-data/power-bi-connector.md:131]: @afloresOSI 
The display looks fine in Zoomin. GitHub just does not display this indent level correctly. 

https://docs.osisoft.com/bundle/ocs/page/visualize-data/power-bi-connector.html

Comment (afloresOSI) [reply content/visualize-data/power-bi-connector.md:131]: Really? It is supposed to be in a code block? The last image isn't showing, it is showing the path in repo.

**Review by afloresOSI — APPROVED** [review]

**Pull request merged**